source/renderer/app/api/voting/requests/getCatalystFund.ts

```diff
@@ -0,0 +1,14 @@
+import { externalRequest } from '../../utils/externalRequest';
+import { MAINNET_SERVICING_STATION_URL } from '../../../config/urlsConfig';
+// import { GetCatalystFundResponse } from '../types';
+
+export const getCatalystFund = (): Promise<string> =>
```

Comment:
I think you've forgotten to update the type to `GetCatalystFundResponse`

Reply:
Right, I'll remove. Thanks!